catalyst_voices/lib/widgets/containers/rails_container.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:catalyst_voices/widgets/containers/space_side_panel.dart';
+import 'package:flutter/material.dart';
+
+/// Widget that creates 'rails' screen structure where two side panels
+/// can be added with main content between them.
+///
+/// |panel |  content  |panel|.
+///
+/// Usually [leftRail] and [rightRail] are used with [SpaceSidePanel].
+///
+/// At the moment we do not have defined behaviour when we don't have
+/// sufficient screen width for all content. Medium and small screens behaviour
+/// will be implemented later. Right now only desktops are focused.
+class RailsContainer extends StatelessWidget {
```

Comment:
How about calling it a `SidebarLayout`?

First of all - wanted to introduce a new category of widgets that just represent different layouts/templates. These templates wouldn't be building blocks like regular widgets but rather act as a whole screen structure. For example `Scaffold` is such structure.

`Container` is not a right word for it because in flutter Container has specific meaning (and it's not a layout/template).

`Sidebar` instead of Rails - sidebar is a more common word than 'Rails'

Reply:
I like idea but `Layout` sounds to me more like smth from android or ios.

How about `SidebarScaffold`? Downside is that it may suggest `Scaffold` inside which maybe dealbreaker, what do you think?

Reply:
Template? Scaffold is okay for me, scaffold is quite close in meaning to layout/template

Reply:
Okey, lets go with `Scaffold`. If it will became confusing ill rename it